src/components/hubble-export.js

```diff
@@ -60,7 +80,7 @@ class HubbleExport extends Component {
         console.log(this.state)
         return (
             <div>
-                <RenderSettingsModal isOpen={this.state.isOpen} handleClose={this.handleClose.bind(this)} />
+                <RenderSettingsModal isOpen={this.state.isOpen} handleClose={this.handleClose.bind(this)} prop={this.props}/>
```

Comment:
Instead of calling this prop "prop", perhaps rename it in RenderSettingsModal, PanelBody, RenderSettingsPanel, and Scene as "mapData" (or something more specific).